src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,93 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final int THIS_IS_ZERO = 0;
+    private static final int THIS_IS_ONE = 1;
+    private static final int THIS_IS_TWO = 2;
+    private T[] currentArray = (T[]) new Object[DEFAULT_CAPACITY];
+    private int size;
+
     @Override
     public void add(T value) {
-
+        checkGrow();
+        currentArray[size++] = value;
     }
 
     @Override
     public void add(T value, int index) {
-
+        size++;
```

Comment:
i think you should increase size in the end

Reply:
hmmm, I wrote this because I got the same comment))

Reply:
agree, it must work correctly